### PR TITLE
OSDOCS#12596: Document the 4.15.38 z-stream release note

### PR DESCRIPTION
**Pull request by tmalove**
Version(s):
4.15

Jira:
[OSDOCS-12596](https://issues.redhat.com/browse/OSDOCS-12596)

Link to docs preview:
[4.15.38](https://84781--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-38_release-notes)

QE review:
- [ ] QE has approved this change.
n/a

Additional information:
The errata URLs will return 404 until the go-live date of 11/13.  Reviewers: I will squash commits after the peer review and before the merge request.

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2773,6 +2773,36 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the Machine Config Operator (MCO) `vSphere `resolv-prepender` script uses `systemd` directives that were not compatible with old boot image versions {product-name} 4. With this release, {product-title} nodes can scale with a boot image 4.13 or later, manual intervention, or an upgrade to a release with this fix. (link:https://issues.redhat.com/browse/OCPBUGS-42110[*OCPBUGS-42110*])
+
+* Previously, when the Image Registry Operator was configured in Azure with `networkAccess:Internal`, you could not successfully set `managementState` to `Removed` in the Operator configuration, This was because an authorization error occurred when the Operator started to delete the storage. With this release, the Operator successfully deletes the storage account, which automatically deletes the storage container. The `managementState` in the Operator configuration is updated to the `Removed` state. (link:https://issues.redhat.com/browse/OCPBUGS-43656[*OCPBUGS-43656*])
```
Comment (lahinson) [inline release_notes/ocp-4-15-release-notes.adoc:2800]:
```suggestion
* Previously, when the Image Registry Operator was configured in {azure-short} with `networkAccess:Internal`, you could not successfully set `managementState` to `Removed` in the Operator configuration. This issue was caused by an authorization error that occurred when the Operator started to delete the storage. With this release, the Operator successfully deletes the storage account, which automatically deletes the storage container. The `managementState` in the Operator configuration is updated to the `Removed` state. (link:https://issues.redhat.com/browse/OCPBUGS-43656[*OCPBUGS-43656*])
```

**Review by lahinson — COMMENTED** [review]
Looks good! Just a few nits. Be sure to squash your commits to 1 when you're finished :)

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -2773,6 +2773,36 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+----
+$ oc adm release info 4.15.38 --pullspecs
+----
+
+[id="ocp-4-15-38-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an invalid or unreachable identity provider (IDP) blocked updates to {hcp}. With this release, the `ValidIDPConfiguration` condition in the `HostedCluster` object now reports any IDP errors and these errors do not block updates of {hcp}. (link:https://issues.redhat.com/browse/OCPBUGS-44201[*OCPBUGS-44201*])
+
+* Previously, the Machine Config Operator (MCO) `vSphere `resolv-prepender` script used `systemd` directives that were not compatible with old boot image versions {product-title} 4. With this release, {product-title} nodes can scale with a boot image 4.13 or later, manual intervention, or an upgrade to a release with this fix. (link:https://issues.redhat.com/browse/OCPBUGS-42110[*OCPBUGS-42110*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-15-release-notes.adoc:2798]: I _think_ this should be:

```suggestion
* Previously, the Machine Config Operator (MCO) vSphere `resolv-prepender` script used systemd directives that were not compatible with old boot image versions of {product-title} 4. With this release, {product-title} nodes can scale with a boot image 4.13 or later, manual intervention, or an upgrade to a release with this fix. (link:https://issues.redhat.com/browse/OCPBUGS-42110[*OCPBUGS-42110*])
```
The final sentence is a little hard to parse to me. Might be that I'm reviewing too early in my day, but double check. :)

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -2773,6 +2773,36 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, when the Image Registry Operator was configured in {azure-short} with `networkAccess:Internal`, you could not successfully set `managementState` to `Removed` in the Operator configuration. This issue was caused by an authorization error that occurred when the Operator started to delete the storage. With this release, the Operator successfully deletes the storage account, which automatically deletes the storage container. The `managementState` in the Operator configuration is updated to the `Removed` state. (link:https://issues.redhat.com/browse/OCPBUGS-43656[*OCPBUGS-43656*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-15-release-notes.adoc:2800]: The backticked terms should generally be functioning as adjectives here. E.g., "The `managementState` property/status in the Operator configuration..." or whatever.

**Review by maxwelldb — COMMENTED** [review]
Couple of quick notes for verification prior to merge.

**Comment by maxwelldb** [comment]
@tmalove I'm still seeing previous issues in the PR, e.g. the formatting error in the second bug fix. Could you verify that the latest and greatest commit is pushed to your remote? Re-label when ready. TY!

**Review by maxwelldb — COMMENTED** [review]
Adding a suggestion for that backtick fix.

**Comment by tmalove** [comment]
Hi @maxwelldb  I made the last change and updated the preview link. Thanks!
/label merge-review-needed
